app/client/components/homePage/conferenceInfo/EventDates.tsx

```diff
@@ -0,0 +1,19 @@
+import { getLocalDateTime } from "@client/utils/time";
+import React, { PropsWithChildren } from "react";
+
+interface EventDatesProps {
+  startDate: Date;
+  endDate: Date;
+  title: string;
+}
+
+export const EventDates = ({ startDate, endDate, title }: EventDatesProps) => {
+  return (
+    <div>
+      <h3>{title}</h3>
```

Comment:
I think you could remove colons at the end of `title` passed and possibly move it here.

Reply:
Done